.changeset/great-needles-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@livekit/components-core": patch
+---
+
+Update USER_CHOICES_KEY to reflect its purpose.
```

Comment:
I think we haven't released the local storage stuff yet, so we don't really need a changeset here, might even be confusing